packages/styles/code.css

```diff
@@ -10,6 +10,14 @@
   --code-border-color: var(--stroke-dark);
 }
 
+.Code__Header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: var(--space-smaller) 0;
```

Comment:
The designs only show an `8px` spacing on the bottom between the header/code block.

Reply:
Added `margin-bottom: var(--space-smallest);` for spacing